### PR TITLE
Adjust scrolling speed for non-Mac desktop OSes 

### LINKED ISSUE #3492: Enhance scroll wheel/touchpad scroll on desktop
### Checklist

- [X] I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same feature, before opening a new one.
- [X] This issue only relates to a single feature. I will open new issues for any other features.

### Is your feature request related to a problem?

The scrolling using mouse wheel (or laptop touchpad) is too slow, comparing to other applications with the same mouse (obviously).

### Is it possible to construct a solution with the existing API?

_No response_

### Describe the solution you'd like to see.

Increase scrolling speed/pages per scroll or any other solution you see is good.

### PR DESCRIPTION
**Pull request by dweymouth**
### Description:
Fresh PR based off of pull #3757, fixing incorrect base branch (now based off of develop), and using conditional compilatino rather than `runtime.GOOS` checks.

Fixes #3492 

### Checklist:

- [ ] Tests included.
- [x] Lint and formatter run with no errors.
- [x] Tests all pass.

### CODE REVIEW

**Comment by dweymouth** [comment]
No idea why the goimports check is failing - maybe someone can take a look?

File: internal/driver/glfw/window.go
Comment (Jacalz) [inline internal/driver/glfw/window.go:22]: This is the reason that `goimports` is failing. With the long names removed from the block, it wants the equal signs to be brought more to the left.

Comment (dweymouth) [reply internal/driver/glfw/window.go]: Yeah, makes sense. Usually VSCode does `gofmt` on save but I think it timed out since it hadn't finished loading the full repo state when I saved the file. Fixed now

File: internal/driver/glfw/scroll_speed_default.go
Comment (Jacalz) [inline internal/driver/glfw/scroll_speed_default.go:1]: Does it not make more sense to call this `_darwin` instead? It seems a bit odd to me that the default file is the one that is only for one operating system.

**Review by Jacalz — CHANGES_REQUESTED** [review]
Thanks for working on this. I just have one minor request for a better name of one of the files.

File: internal/driver/glfw/scroll_speed_default.go
Comment (dweymouth) [reply internal/driver/glfw/scroll_speed_default.go:1]: Is the GLFW driver only used for desktop OSes? I wasn't sure if some other device/platform may be using the GLfW driver

Comment (Jacalz) [reply internal/driver/glfw/scroll_speed_default.go:1]: Yup. The mobile operating systems use the `mobile` driver which is formed from the `gomobile` project. It is only enabled when building with `-tags mobile`.

Comment (dweymouth) [reply internal/driver/glfw/scroll_speed_default.go:1]: I guess the build guards could be simplified then to `darwin` and `!darwin` and the files renamed to `scroll_speed_darwin.go` and `scroll_speed_default.go`. What do you think?

Comment (Jacalz) [reply internal/driver/glfw/scroll_speed_default.go:1]: Sounds good to me

Comment (dweymouth) [reply internal/driver/glfw/scroll_speed_default.go:1]: Done!

**Review by Jacalz — APPROVED** [review]
Thanks. This looks good to me. I'll stand by waiting for a second approval before we merge.

**Pull request merged**
